float32 planning_time
float32 path_length
float32 max_acceleration
float32 execution_time
visualization_msgs/MarkerArray markers
geometry_msgs/PoseArray eef_poses

